spring-kafka-docs/src/main/antora/modules/ROOT/pages/testing.adoc

```diff
@@ -555,12 +528,12 @@ public class KafkaTemplateTests {
     private static final String TEMPLATE_TOPIC = "templateTopic";
 
     @ClassRule
-    public static EmbeddedKafkaRule embeddedKafka = new EmbeddedKafkaRule(1, true, TEMPLATE_TOPIC);
+    public static EmbeddedKafkaBroker embeddedKafka = new EmbeddedKafkaKraftBroker(1, 1, TEMPLATE_TOPIC);
```

Comment:
I'm not sure that this is going to work since `EmbeddedKafkaBroker` is not a JUnit 4 `Rule`.
Pay attention to that `@ClassRule` on this property.

Perhaps `@BeforeClass` can be used instead?